content/200-concepts/100-components/02-prisma-client/140-debugging.mdx

```diff
@@ -17,18 +17,15 @@ You can enable debugging output in Prisma Client via the [`DEBUG`](../../../refe
 Here are examples for setting these debugging options in bash:
 
 ```terminal
-# enable only `prisma:engine`-level debugging output
-export DEBUG="engine"
-
 # enable only `prisma:client`-level debugging output
 export DEBUG="prisma:client"
 
 # enable both `prisma-client`- and `engine`-level debugging output
 export DEBUG="prisma:client,prisma:engine"
 ```
 
-If you want to enable all debugging options, you can also set `DEBUG` to `*`:
+If you want to enable all debugging options, you can also set `DEBUG` to `prisma*`:
```

Comment:
Are we sure all Prisma logging is prefaced with `prisma`?

Reply:
Yes but in old versions it's possible that some were not prefixed.

Reply:
Is there any way to check so that we can give people a range (if you're using Prisma before version X, do this)?

Reply:
In theory yes but that means going through a lot of Git history so I would say maybe we stick with "*" for "all" and let people know that "prisma*" is also a thing to isolate prisma logs. (And avoid babel noise for example)

Reply:
Splendid - that's exactly what I did. Thanks!